### DISCUSSION TITLE
Turn off save image

### DISCUSSION THREAD

**Discussion by Sirmadsen** [lllyasviel/Fooocus#3155] [Q&A] [answered]
Is there anyway to turn off save image automatically? Would be nice to be able to manually save images.

**Comment by mashb1t** [comment, marked as answer]
@Sirmadsen you can achieve that by setting the arg `--disable-image-log` in your startup script (run.bat on Windows or your startup python command/script on Linux)

**Reply by Sirmadsen** [reply]
Thanks!

**Comment by Moorer624** [comment]
While I understand --disable-image-log does effectively stop automatic saving of created images, it's not exactly what I want. When this is arg is included, and I manually download the resulting image, it (a) doesn't save to the right directory, and (b) doesn't put an entry into the log. The ability to direct the saving location and ability to see it in the log, with its metadata would be very helpful.
